executors/dbfixtures/executor.go

```diff
@@ -0,0 +1,134 @@
+package dbfixtures
+
+import (
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/fsamin/go-dump"
+	"github.com/mitchellh/mapstructure"
+	"github.com/ovh/venom"
+	fixtures "gopkg.in/testfixtures.v2"
+
+	// SQL drivers.
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
+)
+
+// Name of the executor.
+const Name = "dbfixtures"
+
+// New returns a new executor that can load
+// database fixtures.
+func New() venom.Executor {
+	return &Executor{}
+}
+
+// Executor is a venom executor that can load
+// fixtures in many databases, using YAML schemas.
+type Executor struct {
+	Files    []string `json:"files" yaml:"files"`
+	Folder   string   `json:"folder" yaml:"folder"`
+	Database string   `json:"database" yaml:"database"`
+	DSN      string   `json:"dsn" yaml:"dsn"`
+	Schema   string   `json:"schema" yaml:"schema"`
+}
+
+// Result represents a step result.
+type Result struct {
+	Executor Executor `json:"executor,omitempty" yaml:"executor,omitempty"`
+}
+
+// Run implements the venom.Executor interface for Executor.
+func (Executor) Run(testCaseContext venom.TestCaseContext, l venom.Logger, step venom.TestStep) (venom.ExecutorResult, error) {
+	// Transform step to Executor instance.
+	var t Executor
+	if err := mapstructure.Decode(step, &t); err != nil {
+		return nil, err
+	}
+	// Connect to the database and ping it.
+	l.Debugf("connecting to database %s, %s\n", t.Database, t.DSN)
+
+	db, err := sql.Open(t.Database, t.DSN)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to database: %v", err)
+	}
+	defer db.Close()
+
+	if err = db.Ping(); err != nil {
```

Comment:
is you one-line the error management (which is good idea) avoid shadow the previous error

Reply:
so, your one-line is useless